tests/puppeteer/package.json

```diff
@@ -23,7 +24,10 @@
   },
   "homepage": "https://github.com/PrestaShop/PrestaShop#readme",
```

Comment:
Maybe change the homePage to https://github.com/PrestaShop/PrestaShop/tree/develop/tests/puppeteer#readme

Reply:
Done, thanks !